docs/docs/external-volumes.md

```diff
@@ -84,7 +84,7 @@ You can specify an external volume in your Marathon app definition. [Learn more
       "instances": 1,
       "cpus": 0.1,
       "mem": 32,
-      "cmd": "/usr/bin/tail -f /dev/null",
+      "cmd": "mkdir my-test-vol && touch my-test-vol/test.txt && ls my-test-vol/",
```

Comment:
....I still get the error, but maybe it's implied that a user already has a directory called `my-test-vol`?

from STDERR:
```
touch: cannot touch 'my-test-vol/test.txt': No such file or directory
```